packages/apis/ipfs/src/query/schema.graphql

```diff
@@ -0,0 +1,6 @@
+#import { Query, ResponseType } into Http from "w3://ens/http.web3api.eth"
+
+type Query {
+  catToString(ipfsUrl: String!, cid: String!): String!
```

Comment:
I think we can remove this, as it's a nice helper, but doesn't exist in IPFS's official HTTP API.